llama_index/evaluation/benchmarks/beir.py

```diff
@@ -0,0 +1,87 @@
+from typing import Callable, List, Dict
+from llama_index.utils import get_cache_dir
+from llama_index.schema import Document
+from llama_index.indices.base_retriever import BaseRetriever
+import os
+import tqdm
+
+beir_datasets = [
+    "trec-covid",
+    "hotpotqa",
```

Comment:
add the others?

Perhaps link to the repo as well showing the sizes of the various corpuses... Most are pretty large > 10K